charts/operator/templates/collector.yaml

```diff
@@ -178,6 +178,191 @@ spec:
                 operator: In
                 values:
                 - linux
+              - key: cloud.google.com/gke-monitoring-variant
+                operator: NotIn
+                values:
+                - MAX_THROUGHPUT
+      tolerations:
+      - effect: NoExecute
+        operator: Exists
+      - effect: NoSchedule
+        operator: Exists
+      securityContext:
+        runAsGroup: 1000
+        runAsNonRoot: true
+        runAsUser: 1000
+        seccompProfile:
+          type: RuntimeDefault
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: max-collector
+  namespace: {{.Values.namespace.system}}
+  {{- if .Values.commonLabels }}
+  labels:
+    {{- include "prometheus-engine.collector.labels" . | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      # DO NOT MODIFY - label selectors are immutable by the Kubernetes API.
+      # see: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#pod-selector.
+      {{- include "prometheus-engine.collector.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "prometheus-engine.collector.templateLabels" . | nindent 8 }}
+      annotations:
+        # The emptyDir for the storage and config directories prevents cluster
+        # autoscaling unless this annotation is set.
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+        components.gke.io/component-name: managed_prometheus
+    spec:
+      serviceAccountName: {{ include "prometheus-engine.collector.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      priorityClassName: gmp-critical
+      initContainers:
+      - name: config-init
+        image: {{.Values.images.bash.image}}:{{.Values.images.bash.tag}}
+        command: ['/bin/bash', '-c', 'touch /prometheus/config_out/config.yaml']
+        volumeMounts:
+        - name: config-out
+          mountPath: /prometheus/config_out
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - all
+          privileged: false
+          readOnlyRootFilesystem: true
+      containers:
+      - name: config-reloader
+        image: {{.Values.images.configReloader.image}}:{{.Values.images.configReloader.tag}}
+        args:
+        - --config-file=/prometheus/config/config.yaml
+        - --config-file-output=/prometheus/config_out/config.yaml
+        - --reload-url=http://127.0.0.1:19090/-/reload
+        - --ready-url=http://127.0.0.1:19090/-/ready
+        - --listen-address=:19091
+        ports:
+        - name: cfg-rel-metrics
+          containerPort: 19091
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        resources: {{- toYaml $.Values.resources.bash | nindent 10}}
```

Comment:
suggestion: shouldn't this be `resources.configReloader`? Same with L76.

Reply:
Yup. Looks like that was incorrect before. Good catch!